test/e2e/problemmaker/README.md

```diff
@@ -0,0 +1,18 @@
+# Problem Maker
+
+Problem maker is a program to generate/simulate various kinds of node problems. It is used in NPD e2e tests to verify NPD's behavior when node problems happen:
+1. NPD should report the problems correctly.
+2. NPD should survive the problems as much as possible.
+
+Problem maker is NOT intended to be used in any other places. And please do NOT run this directly on your workstation, as it can cause real node failures. You shouldn't need to run it anyways. If you want to test NPD, it's best to run NPD e2e test.
```

Comment:
Maybe highlight the part about not running on workstation, which can cause problems.

And probably give an example here, e.g., making the disk readonly, which can only be recovered by reboot.

Reply:
Sounds good :)
Stressed the warning and provided the example.